app/presentation/RoomItem/LastMessage.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Text } from 'react-native';
+import { emojify } from 'react-emojione';
+import PropTypes from 'prop-types';
+
+import I18n from '../../i18n';
+import styles from './styles';
+
+const formatMsg = ({
+	lastMessage, type, showLastMessage, username
+}) => {
+	if (!showLastMessage) {
+		return '';
+	}
+	if (!lastMessage) {
+		return I18n.t('No_Message');
+	}
+
+	let prefix = '';
+	const me = lastMessage.u.username === username;
+
+	if (!lastMessage.msg && Object.keys(lastMessage.attachments).length > 0) {
+		if (me) {
+			return I18n.t('User_sent_an_attachment', { user: I18n.t('You') });
+		} else {
+			return I18n.t('User_sent_an_attachment', { user: lastMessage.u.username });
```

Comment:
instead of duplicating `return I18n.t('User_sent_an_attachment', { user: ` declare `user` variable and assign value to it in if\else block. And after if\else block: `return I18n.t('User_sent_an_attachment', { user });`